website/src/content/docs/docs/emitters/clients/http-client-java/reference/emitter.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Emitter usage"
+---
+
+## Usage
+
+1. Via the command line
+
+```bash
+tsp compile . --emit=@typespec/http-client-java
+```
+
+2. Via the config
+
+```yaml
+emit:
+  - "@typespec/http-client-java"
+```
+
+The config can be extended with options as follows:
+
+```yaml
+emit:
+  - "@typespec/http-client-java"
+options:
+  "@typespec/http-client-java":
+    option: value
+```
+
+## Emitter options
+
+### `namespace`
+
+**Type:** `string`
+
+Java package/namespace. If not provided, emitter would use package name converted from TypeSpec namespace.
+
```

Comment:
Should we remove this section now given `EmitterOptions` no longer supports `namespace` ([pr](https://github.com/microsoft/typespec/pull/6546))?

Reply:
Yes, I need to refresh main and regen the docs in this PR.